cli/lint.rs

```diff
@@ -200,16 +208,26 @@ struct LintError {
 
 struct PrettyLintReporter {
   lint_count: u32,
+  check_count: u32,
+  verbose: bool,
 }
 
 impl PrettyLintReporter {
-  fn new() -> PrettyLintReporter {
-    PrettyLintReporter { lint_count: 0 }
+  fn new(verbose: bool) -> PrettyLintReporter {
+    PrettyLintReporter {
+      lint_count: 0,
+      check_count: 0,
+      verbose,
+    }
   }
 }
 
 impl LintReporter for PrettyLintReporter {
-  fn visit(&mut self, d: &LintDiagnostic, source_lines: Vec<&str>) {
+  fn visit_file(&mut self) {
+    self.check_count += 1;
```

Comment:
Maybe pass number of checked files to `.close()` method?

Reply:
I fixed `LintReporter` interface to make it simple.
`.close` receives the number of checked files and `verbose` now, so `visit_file` and `should_report_verbosely` are removed.